net/newsraft/Portfile

```diff
@@ -27,4 +27,14 @@ depends_lib-append  port:curl \
                     port:sqlite3 \
                     port:yajl
 
+# Otherwise linking to LegacySupport is not done,
+# and build fails on systems where it is needed.
+patchfiles-append   patch-makefile.diff
+
+post-patch {
+    reinplace "s|@CC@|${configure.cc}|" ${worksrcpath}/makefile
+    reinplace "s|@CFLAGS@|${configure.cflags} [get_canonical_archflags cc]|" ${worksrcpath}/makefile
+    reinplace "s|@LDFLAGS@|${configure.ldflags} [get_canonical_archflags ld]|" ${worksrcpath}/makefile
+}
```

Comment:
Don't patch; use `makefile.override-append CFLAGS LDFLAGS`. (`CC` is already in the default value of `makefile.override`).

Reply:
Did not work for me, but I dropped patching CC value, that is unneeded.

Reply:
Did you try debugging the failure? It should have worked.

Reply:
As long as it is a matter of aesthetics, not functionality, I rather not spend an extra hour on trying to force this to work, when patching fixes the problem (and is done in multiple other ports in similar cases).